index.html

```diff
@@ -0,0 +1,524 @@
+<!DOCTYPE html><html><head><link href="https://fonts.googleapis.com/css?family=Inter&display=swap" rel="stylesheet" /></head><body><div class="v5_3"><div class="name"></div><div class="v5_5"></div><div class="v5_6"><div class="v5_7"></div><span class="v5_8">What language is used to create databases?</span></div><div class="v5_9"><div class="v5_10"></div><span class="v5_11">1z3</span></div><div class="v5_12"><div class="v5_13"></div><span class="v5_14">00:30</span></div><div class="v5_15"><div class="v5_16"><div class="v5_17"></div><span class="v5_18">C++</span></div><div class="v5_19"><div class="v5_20"></div><span class="v5_21">C++</span></div><div class="v5_22"><div class="v5_23"></div><span class="v5_24">C++</span></div><div class="v5_25"><div class="v5_26"></div><span class="v5_27">C++</span></div></div><div class="v5_28"><div class="v5_29"><div class="v5_30"></div><span class="v5_31">C++</span></div><div class="v5_32"><div class="v5_33"></div><span class="v5_34">C++</span></div><div class="v5_35"><div class="v5_36"></div><span class="v5_37">C++</span></div><div class="v5_38"><div class="v5_39"></div><span class="v5_40">C++</span></div></div><span class="v5_41">Player 1</span><span class="v5_42">Player 2</span></div></body></html> <br/><br/> <style>* {
```

Comment:
Nebudem tu pisat detaily, lebo nie som si isty, ci ide o final verziu z tvojej strany alebo je to len nejaka pilotna verzia, ktoru budes upravovat ...kazdopadne par komentov prenecham po kolegoch, co by ti mohli nejakym sposobom pretlmocit, lebo keby to tu napisem tak by asi prisli dalsie otazky....


co sa tyka zakladnej chyby, ktoru tu vidim, bezohladu na to ci je toto len draft commit index.html (aj pre index2.html) je to, ze subor HTML musi byt jasne citatelny...to znamena formatovanie je velmi dolezite v hocijakom jazyku (java, html, javascript, python) vsade su nejake zakladne odsadenia, nove riadky atd ...

dalsia vec co je velmi velmi zla v tomto pripade je `<style> </style>` v kazdom subore ...pokial sa riesi styl, pre viacero stranok (podstranok) vzdy sa odporuca vyuzivat stylovanie pomocou `.css` suboru, ktory sa importne do kazdeho .`.html` suboru

samotne classes , co vygenerovala ta figma, su tiez velmi velmi zle, malo kedy pocas vyvoja webu/webovej stranky sa stretnes s tym, ze kazde `<div> alebo <span> alebo <p>` bude mat definovany vlastny styl (vynimka su margin/padding)